src/diffusers/utils/testing_utils.py

```diff
@@ -144,6 +152,23 @@ def floats_tensor(shape, scale=1.0, rng=None, name=None):
     return torch.tensor(data=values, dtype=torch.float).view(shape).contiguous()
 
 
+def is_staging_test(test_case):
```

Comment:
Why is it defined in `src/diffusers/utils/testing_utils.py` and not in the `tests/` folder directly? I understand that this is the way it is done in `transformers` but I'm not sure to get the real underlying reason.
Having it in `tests/` would remove the problem of having `pytest` as a soft dependency in the implementation.

Reply:
`transformers`